### PR TITLE
Yes 65 bottom line inline styles

### PR DESCRIPTION
**Pull request by ijazalise**
YES 65 done.

### CODE REVIEW

File: view/frontend/layout/catalog_category_view.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!--
+/**
+* @copyright Copyright (c) 2016 www.innovadeltech.com
+*/
+-->
```
Comment (DanGlz) [inline view/frontend/layout/catalog_category_view.xml]: please remove this comment

File: view/frontend/layout/catalog_product_view.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!--
+/**
+* @copyright Copyright (c) 2016 www.innovadeltech.com
+*/
+-->
```
Comment (DanGlz) [inline view/frontend/layout/catalog_product_view.xml]: please remove this comment

File: view/frontend/layout/cms_index_index.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<!--
+/**
+* @copyright Copyright (c) 2016 www.innovadeltech.com
+*/
+-->
```
Comment (DanGlz) [inline view/frontend/layout/cms_index_index.xml]: please remove this comment

File: view/frontend/layout/default.xml [hunk 4]
```diff
@@ -1,6 +1,7 @@
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../../../../../lib/internal/Magento/Framework/View/Layout/etc/page_configuration.xsd">
-    <body>
 
+    <body>
+
```
Comment (DanGlz) [inline view/frontend/layout/default.xml]: please remove empty lines

File: view/frontend/templates/bottomline.phtml [hunk 5]
```diff
@@ -1,15 +1,6 @@
...
-<style>
-    .yotpo-icon-double-bubble, .yotpo-stars { margin: 0 5px 0 0 !important; }
-    div.yotpo.bottomLine { display: inline-block; margin: 0 10px 1px 0;}
-    .thumbnail .yotpo a { display: inline-block; }
-    div.yotpo.QABottomLine { display: inline-block; }
-    .product-info-main .product-reviews-summary{width:100%;}
-</style>
-<?php } ?>
-
+<div class="yotpoBottomLine">
```
Comment (DanGlz) [inline view/frontend/templates/bottomline.phtml]: what it the purpose of this div? can we remove it?

Comment (ijazalise) [reply view/frontend/templates/bottomline.phtml]: This div helps to keep bottomline and questions answers block in a row.

Comment (ijazalise) [reply view/frontend/templates/bottomline.phtml]: This div helps to keep bottomline and questions answers block in a row.

Comment (ijazalise) [reply view/frontend/templates/bottomline.phtml]: All other required changes have been done.

**Pull request merged**